client/my-sites/importer/importer-header.jsx

```diff
@@ -102,3 +114,9 @@ export default React.createClass( {
 		);
 	}
 } );
+
+const mapDispatchToProps = dispatch => ( {
+	startImport: flowRight( dispatch, startImport )
+} );
+
+export default connectDispatcher( null, mapDispatchToProps )( ImporterHeader );
```

Comment:
Oops - haha. This isn't `connect` - it's my transition function which doesn't have that property. Reverting the changes…